stage1/init/init.go

```diff
@@ -215,6 +216,18 @@ func installAssets() error {
 	return proj2aci.PrepareAssets(assets, "./stage1/rootfs/", nil)
 }
 
+// supportsNotify return true if in the pod manifest appc.io/executor/supports-systemd-notify is set to true
+func supportsNotify(p *stage1commontypes.Pod) bool {
```

Comment:
dry with stage1/init/common#supportsNotify, just make the first one public, it has no side-effects.